MaraBot/Core/Option.cs

```diff
@@ -0,0 +1,84 @@
+using System.ComponentModel.DataAnnotations;
+using System.Collections.Generic;
+
+namespace MaraBot.Core
+{
+	/**
+	 * Randomizer modes the options are categorized by.
+	 */
```

Comment:
For C# code documentation, I would favor XML Doc instead:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments

```suggestion
	/// <summary>
	/// Randomizer modes the options are categorized by.
	/// </summary>
```